src/mumble/AudioInput.cpp

```diff
@@ -384,6 +384,14 @@ void AudioInput::addMic(const void *data, unsigned int nsamp) {
 					if (qlEchoFrames.isEmpty()) {
 						iJitterSeq = 0;
 						iMinBuffered = 1000;
+					} else if (qlEchoFrames.count() >= 100) {
```

Comment:
I was hoping you would understand it ;) Imho the code is fishy. My guess is what I said above is probably the intention (detecting prolonged overfilled buffer situations and correcting them by discarding frames) but to me it looks convoluted and broken w.r.t to actually achieving that goal (as you said, the buffer is bound to be low after a reset and we still adjust iMin, what's the information in that? It probably only triggers in severly skewed scenarios where it can't help anyways). If we agree on not understanding we should at least mark it with a "FIXME" note and open an issue. We'll probably want to rewrite or discard that code going forward if we cannot figure out some brilliant working principle we've just overlooked....